### PR TITLE
Update documentation for `schema.js`

### PR DESCRIPTION
**Pull request by corrspt**
I had an issue with findBy, so I updated the docs a little bit.

### CODE REVIEW

File: lib/orm/schema.js [hunk 1]
```diff
@@ -256,10 +256,16 @@ export default class Schema {
 
   /**
     Returns the first model in the database that matches the key-value pairs in `attrs`. Note that a string comparison is used.
+    Also remeber that attribute names are dasherized (for attributes with more than 1 word, like "createdAt" becomes "created-at")
```
Comment (samselikoff) [inline lib/orm/schema.js]: Hm – I don't think this is true. I think maybe somewhere else a dasherized compound word must have been used.

Could you try with a new app that uses a model with a compound attribute name, and see if it's the case?

Comment (corrspt) [reply lib/orm/schema.js]: Hi Sam,

I'll do that. Might take me a little while to setup and test that, but I'll get back when I do!

Comment (asantos00) [inline lib/orm/schema.js]: Minor: Typo in `remember`

File: lib/orm/schema.js [hunk 2]
```diff
@@ -256,10 +256,16 @@ export default class Schema {
...
 
     ```js
     let post = blogPosts.findBy({ published: true });
     ```
+    
+    Remember that relantionships are represented with a camelized `relationNameid` attribute name, so if you want to search for an author, you would do:
```
Comment (asantos00) [inline lib/orm/schema.js]: typo: `relationships`

File: lib/orm/schema.js (hunk 1 above)
Comment (corrspt) [reply lib/orm/schema.js]: Thanks Alexandre for the typo report

**Comment by samselikoff** [comment]
Simplified this a bit by just adding some more examples. Thanks @corrspt!

**Comment by corrspt** [comment]
Thanks so much for moving this forward @samselikoff. I wanted to contribute, but couldn't find the time again.

**Pull request merged**